io.itch.itch.yaml

```diff
@@ -0,0 +1,133 @@
+app-id: io.itch.itch
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.winehq.Wine
+base-version: stable-22.08
+command: itch-run
+separate-locales: false
+finish-args:
+  - --allow=devel # For Wine crash handling
```

Comment:
Given that anyone can upload anything to itch, this is quite dangerous.

Reply:
The itch ovh is their official download CDN, and due to there being currently no known way to automatically check for an update, its quite secure as a human (me) has to manually verify the build.

Reply:
I mean, everyone can submit games, which itch then executes, and you essentially disabled the @flatpak sandbox. This setting is used exclusively for debuggers so that developers can attach to processes. It is dangerous.

Reply:
ah, now i get what you mean, let me chop that off